packages/metals-languageclient/src/getJavaHome.ts

```diff
@@ -9,64 +9,89 @@ import {
   IJavaHomeInfo,
 } from "@viperproject/locate-java-home/js/es5/lib/interfaces";
 import { toPromise } from "./util";
-//import os from "os";
 import fs from "fs";
 import path from "path";
+import { spawn } from "promisify-child-process";
+
+export type JavaVersion = "8" | "11" | "17" | "21";
```

Comment:
After https://github.com/scalameta/metals/pull/6092 we will be able to leave 17 and 21